manifests/init.pp

```diff
@@ -51,14 +51,21 @@
     ensure => $package_ensure,
   }
 
+  $ensure_status = $enable ? {
+    0       => absent,
+    1       => present,
+    default => present
+  }
```

Comment:
```suggestion
   $ensure_status = $enable ? { 0 => 'absent',  default  => 'present' }
```

Maybe worthwhile to redefine $enable as `Variant[Integer[0, 1],Boolean]` and mark the 0/1 as deprectaed?